plugins/inputs/googlecoreiot/README.md

```diff
@@ -0,0 +1,69 @@
+# Google Core IoT service input plugin
+
+The Google Core IoT listener is a service input plugin that listens for messages sent via HTTP POST from Google's Pub/Sub service.
```

Comment:
Add link to associated service documentation.  I'm wondering what makes this a Google Core IoT plugin instead of a general pub/sub consumer, can you explain the relationship between these technologies?  Whats the scoop on Google Core IoT vs Google Cloud IoT?

Reply:
Please speak to Sharjeel and Russ about this.